std_msgs/Int64 agent_i
std_msgs/Int64 agent_j
geometry_msgs/Point r_ij_rel_i
